packages/react-native/src/Authenticator/Defaults/ConfirmResetPassword/ConfirmResetPassword.tsx

```diff
@@ -41,23 +37,26 @@ const ConfirmResetPassword: DefaultConfirmResetPasswordComponent = ({
     handleSubmit,
   });
 
+  const headerText = getResetYourPasswordText();
+  const primaryButtonText = isPending ? getSubmittingText() : getSubmitText();
+  const secondaryButtonText = getResendCodeText();
+
+  const buttons = useMemo(
+    () => ({
+      primary: { children: primaryButtonText, onPress: handleFormSubmit },
+      links: [{ children: secondaryButtonText, onPress: resendCode }],
```

Comment:
Should this be the secondary button since `resendCode` doesn't navigate the user to a different subcomponent?